app/scripts/views/components/dataTable/tableView.js

```diff
@@ -227,11 +227,15 @@
       }
       return assignedColors[key];
     }
+
     function initPieTableData() {
       _.each(group.all(), function(attr, index) {
+        if (!attr.value || attr.value === 'na') {
```

Comment:
Instead of checking it here why don't we do it at the time of creating dimension.